Microsoft.Toolkit.Uwp.UI.Controls/AdMediator/UwpAdMediator.xaml.cs

```diff
@@ -0,0 +1,377 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    using Microsoft.Advertising.WinRT.UI;
+#pragma warning disable SA1208 // System using directives must be placed before other using directives
+    using System;
+#pragma warning restore SA1208 // System using directives must be placed before other using directives
+    using Windows.System.Profile;
+    using Windows.UI.Xaml;
+    using Windows.UI.Xaml.Controls;
+
+    // This control written per https://msdn.microsoft.com/windows/uwp/monetize/migrate-from-admediatorcontrol-to-adcontrol
+
+    /// <summary>A UWP AdMediator control</summary>
+    /// <seealso cref="Windows.UI.Xaml.Controls.UserControl" />
+    /// <seealso cref="Windows.UI.Xaml.Markup.IComponentConnector" />
+    /// <seealso cref="Windows.UI.Xaml.Markup.IComponentConnector2" />
+    public sealed partial class UwpAdMediator : UserControl
+    {
+        public int MobileAdWidth
```

Comment:
Maybe instead of all of the Mobile/Desktop values, allow the user to decide which ad size/Id they wish to use within their app by using their own state triggers. They could decide when to change. This does add  some completely that the user must have state triggers (or listen to size changed) compared to settings multiple values